packages/commonwealth/scripts/refresh-all-memberships.ts

```diff
@@ -1,26 +1,35 @@
 import Bluebird from 'bluebird';
-import { TokenBalanceCache } from '../../token-balance-cache/src';
+import { RedisCache } from '../../common-common/src/redisCache';
+import { TokenBalanceCache as TokenBalanceCacheV1 } from '../../token-balance-cache/src';
+import { REDIS_URL, VULTR_IP } from '../server/config';
 import { ServerCommunitiesController } from '../server/controllers/server_communities_controller';
 import { ServerGroupsController } from '../server/controllers/server_groups_controller';
 import db from '../server/database';
 import BanCache from '../server/util/banCheckCache';
+import { TokenBalanceCache as TokenBalanceCacheV2 } from '../server/util/tokenBalanceCache/tokenBalanceCache';
 
 async function main() {
   const models = db;
-  const tokenBalanceCache = new TokenBalanceCache();
-  await tokenBalanceCache.initBalanceProviders();
-  await tokenBalanceCache.start();
+  const redisCache = new RedisCache();
+  await redisCache.init(REDIS_URL, VULTR_IP);
```

Comment:
No need to pass VULTR_IP. Passing VULTR_IP alters the behavior of the controller so it shouldn't be passed. This needs to be deprecated since we don't use VULTR anymore. Just created this issue: https://github.com/hicommonwealth/commonwealth/issues/5976
```suggestion
  await redisCache.init(REDIS_URL);
```

Reply:
good to know– applied